packages/react-native/React/CoreModules/RCTAlertController.mm

```diff
@@ -20,17 +20,7 @@ @implementation RCTAlertController
 - (UIWindow *)alertWindow
 {
   if (_alertWindow == nil) {
-    _alertWindow = [self getUIWindowFromScene];
-
-    if (_alertWindow == nil) {
-      UIWindow *keyWindow = RCTSharedApplication().keyWindow;
-      if (keyWindow) {
-        _alertWindow = [[UIWindow alloc] initWithFrame:keyWindow.bounds];
-      } else {
-        // keyWindow is nil, so we cannot create and initialize _alertWindow
-        NSLog(@"Unable to create alert window: keyWindow is nil");
-      }
-    }
+    _alertWindow = [[UIWindow alloc] initWithWindowScene:RCTKeyWindow().windowScene];
```

Comment:
The previous check was also checking that the scene activation state was `active`.
can the `windowScene` retrieved by `RCTKeyWindow` be `inactive` (e.g.: there is already another alert presented)?

Reply:
I've updated the function to take the activation state into account. As, we should always get the currently focused window. With the previous approach it was possible to retrieve some inactive keyWindow of additional scene (for example created by the OS). Can you check if the internal tests are passing now?